### DISCUSSION TITLE
Struggling

### DISCUSSION THREAD

**Discussion by janemiceli** [YU000jp/logseq-plugin-weekdays-and-weekends#37] [General]
I'm sturgging to make this plugin work.

Picture below. 

The template doesn't render.

**Comment by janemiceli** [comment]
![image](https://github.com/YU000jp/logseq-plugin-weekdays-and-weekends/assets/1576764/bf0f8569-be55-4fb4-be94-8c5440a65679)

![image](https://github.com/YU000jp/logseq-plugin-weekdays-and-weekends/assets/1576764/bc61cac5-dcdf-486c-ac37-96de13d345bd)

![image](https://github.com/YU000jp/logseq-plugin-weekdays-and-weekends/assets/1576764/93df6260-8935-460b-93af-1e8ee07ce7ec)

**Reply by YU000jp** [reply]
`:default-template` -> `:default-templates`
It could be a spelling mistake.
